.github/workflows/release.yml

```diff
@@ -105,26 +106,11 @@ jobs:
         with:
           myToken: ${{ secrets.ACCESS_TOKEN }}
 
-      - name: Create Release
-        id: create_release
-        uses: actions/create-release@latest
-        env:
-          GITHUB_TOKEN: ${{ secrets.ACCESS_TOKEN }}
+      - name: Release
+        uses: softprops/action-gh-release@v1
```

Comment:
Why switch release tool? Unnecessary. prefer actions that are GH maintained if we've already got that in play.